test/blackjack_test.rb

```diff
@@ -11,6 +11,7 @@ def setup
     @large_hand = [%w(9 diamonds), %w(10 diamonds)]
     @small_ace_hand = [%w(10 diamonds), %w(5 hearts), %w(ace spades)]
     @large_ace_hand = [%w(3 diamonds), %w(5 hearts), %w(ace spades)]
+    @triple_aces_hand = [%w(ace spades), %w(ace hearts), %w(ace clubs)]
   end
 
```

Comment:
Very rare condition to have multiple aces in a single deck, but not out of the question, and a lot of students who write their own method to calculate the total miss this one when all the other tests succeed.